Assets/Game/Addons/LocationEditor/Editor/LocationEditor.cs

```diff
@@ -0,0 +1,537 @@
+using System.Collections.Generic;
+using UnityEditor;
+using UnityEngine;
+using DaggerfallWorkshop.Utility.AssetInjection;
+using System.IO;
+
+/// <summary>
+/// Location editor
+/// </summary>
+namespace DaggerfallWorkshop.Loc {
+
+    public class LocationEditor : EditorWindow {
```

Comment:
I would suggest not to use an abbreviation for the namespace. The summary should also be placed above the class definiton, not the namespace.